### DISCUSSION TITLE
Table builder inside action modal

### DISCUSSION THREAD

**Discussion by aymanalareqi** [filamentphp/filament#10471] [Help]
is there a way to show table inside Action Modal?

**Comment by leandrocfe** [comment]
You could show it using the `->modalContent()`

- Create a livewire component and [add a table](https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component)
- Create a view `php artisan make:view custom-table`
- Render the livewire component (table) in the view 

```php
<div>
@livewire('my-table')
</div>
```
- Add the view in the modal `->modalContent(view('custom-table'))`
